### ISSUE TITLE
Better font for formulas

### ISSUE THREAD

**Issue by lakesare** [lakesare/memcode#58]
![image](https://user-images.githubusercontent.com/7578559/77834803-3866fe00-7169-11ea-856c-ea8da78b0a7e.png)
https://www.memcode.com/courses/1410

Formulas look ugly right now. Is it possible to use a better font for formulas in Quill's Katex extension?

**Comment by Pavithran12** [comment]
Hello, I had a look into this particular page and more than fonts, I have a thing to add up which I feel may causing certain issues in the quality of it.
![image](https://user-images.githubusercontent.com/20036464/78162960-42d51080-7465-11ea-89e9-b20f29e637a4.png)
If you see the images the formula size variance in the font is mostly because of the type of equation we use. In formula 35 the division is represented in 8/8 but when it comes near an equal symbol it changes to another format that is in formula 36.
Do you feel changing the font alone will help us improve the way the formula's look?

**Comment by Pavithran12** [comment]
I would also like to add that there is considerable difference between the font size of the formulas from one to other also makes them look clumsy. I think there should be a standardization overall at the class level.

**Comment by lakesare** [comment]
I agree that part of the awkwardness is the font-size variance (first formula in flashcard 36 shows it well).

It seems to be:
- different font-sizes
- italic text (instead of normal font-style)
- font weight (too heavy)

`.katex{ font-style: normal; font-size: 20px; }` seems to improve it.
Removing `font-family: KaTeX_Math;` (from `.mathdefault` selectors e.g.) seems to make it better too.

Before:
![image](https://user-images.githubusercontent.com/7578559/78165708-2df87d00-7465-11ea-89b6-cf6c28ddffe9.png)

After:
![image](https://user-images.githubusercontent.com/7578559/78165593-fbe71b00-7464-11ea-9145-acc76e247532.png)

**Comment by lakesare** [comment]
I think we can't easily make the font consistent, because katex sets `font-size: 0.7em` on purpose.
There could be some better plugin/some better font for Quill that cares about consistent font-size more.

**Comment by lakesare** [comment]
Created the related issue https://github.com/quilljs/quill/issues/3116.
